oshi-core/src/main/java/oshi/hardware/NetworkIF.java

```diff
@@ -79,6 +79,15 @@ default String getIfAlias() {
         return "";
     }
 
+    /**
+     * The {@code ifOperStatus} as described in RFC 2863.
```

Comment:
Add some caveats about Windows and Linux only, similar to the `getIfAlias()` ones.